device/rsvp/rsvpLspInterface.yaml

```diff
@@ -0,0 +1,27 @@
+components:
+  schemas:
+    Rsvp.LspInterface:
+      x-status: under-review
+      description: >-
+        Configuration for RSVP LSP Interface.
+      type: object
+      required: [ipv4_name]
+      properties:
+        ipv4_name: 
+          description: >-
+            The globally unique name of the IPv4 or Loopback IPv4 interface acting as the RSVP ingress and egress endpoint for 
+            the LSPs configured on this interface.
```

Comment:
The description looks unclear to me. Cant it like - "This should be the "name" field of either ipv4 or ipv4 loopback interface that you want to make ingress or egress for your LSPs on this RSVP interface"

Reply:
This is a generic comment - I think if you try to write the documentation in the second person that will less cryptic.

Reply:
Added further clarification on exact match needed with "name" field of "ipv4_interface" or "ipv4_loopback." . But for now not modifying to second person ( your...) . This is because in vendor documentation or openconfig/yang model documentation, we could not find any such usage.